docs/Security-Compliance/Security-Controls/AC/AC-05-Separation-of-duties.md

```diff
@@ -0,0 +1,62 @@
+# Access Control
+## AC-05 - Separation of Duties
+
+The organization:  
+a. Separates [at a minimum, data creation and control, software development and maintenance, and security functions];
+
+b. Documents separation of duties of individuals; and 
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+Software development and maintenance can be accomplished by a user that has a *superuser* and *staff* flag in the Django Admin  Console (DAC). This also allows the user to login to the DAC and complete all CRUD functions. System owner, also has *superuser* and *staff* flag, and manages the account access. Additionally, System owner provisions a new user account on request.
```

Comment:
> Software development and maintenance can be accomplished by a user that has a *superuser* and *staff* flag in the Django Admin  Console (DAC). 

This doesn't seem accurate to me -- I think we are using tools like GitHub, CircleCI, and Cloud.gov for software development and _maintenance,_ not the Django Admin console.

Reply:
We want to just point out how each tool is used to separate the two functions/activities of software dev and maintenance, correct?

Reply:
I think more like -- how & where do the devs do their dev work? How & where do the folks who manage data, approvals do their work?

Reply:
See also: https://github.com/HHS/TANF-app/pull/125#issuecomment-760457973